fdbserver/MutationTracking.cpp

```diff
@@ -0,0 +1,100 @@
+/*
+ * MutationTracking.cpp
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2020 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <vector>
+#include "fdbserver/MutationTracking.h"
+#include "fdbserver/LogProtocolMessage.h"
+
+#if defined(FDB_CLEAN_BUILD) && MUTATION_TRACKING_ENABLED
+#error "You cannot use mutation tracking in a clean/release build."
+#endif
+
+StringRef debugKey = LiteralStringRef( "\xff/globals/lastEpochEnd" );
```

Comment:
This was `LiteralStringRef( "" );` in the original code. Maybe we should keep using it?

And it would be nice to add some comment for these two keys so that if a developer need to look into specific version/key/range, he/she knows how to use these two keys.